package.json

```diff
@@ -102,7 +102,6 @@
     "webpack-dev-server": "^4.8.1"
   },
   "resolutions": {
-    "@carbon/react": "1.17.0",
```

Comment:
I may not have full context on what's happening here, why was this removed?

Reply:
Its an older version and is used by default